### PR TITLE
Adding a Running Tests pages

### PR DESCRIPTION
**Pull request by kdhamric**
Adding a Running Tests pages in docs in CLI section

This PR adds a page to the documentation that discusses various ways to run a test.

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by mathnogueira — APPROVED** [review]
LGTM!

File: docs/docs/cli/running-tests.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+            }
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+You can also opt to output the result as junit to a file. You would run the command with a -j option and a file name, ie:
```
Comment (jfermi) [inline docs/docs/cli/running-tests.md:165]:
```suggestion
You can also opt to output the result as JUnit to a file. You would run the command with a -j option and a file name, ie:
```

File: docs/docs/cli/running-tests.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+```
+tracetest test run -d path/to/test.yaml -w -j junit.out
+```
+The junit output file would then contain the junit result, for example:
```
Comment (jfermi) [inline docs/docs/cli/running-tests.md:169]:
```suggestion
The JUnit output file would then contain the JUnit result, for example:
```

File: docs/docs/cli/running-tests.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+                <testcase name="attr:tracetest.span.duration &lt; 20ms"></testcase>
+        </testsuite>
+</testsuites>
+```
+
+## Running a Test That Uses Environment Variables
+
+There are two ways of referencing an environment when running a test. 
+
+You can reference an existing environment using it's id by running. For example, given this defined environment with an id of 'testenv':
```
Comment (jfermi) [inline docs/docs/cli/running-tests.md:192]:
```suggestion
You can reference an existing environment using its id. For example, given this defined environment with an id of 'testenv':
```

File: docs/docs/cli/running-tests.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+![testenv](../img/show-environment-definition.png)
+
+We can run a test and specify that environment with this command:
+```
+tracetest test run -d path/to/test.yaml -e testenv -w
+```
+
+You can also reference an .env file which will be used to create a new environment or updating an existing one. For example, if you have a .env file named local.env with this content:
```
Comment (jfermi) [inline docs/docs/cli/running-tests.md:201]:
```suggestion
You can also reference an .env file which will be used to create a new environment or update an existing one. For example, if you have a .env file named local.env with this content:
```

File: docs/docs/cli/running-tests.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+```
+POKEID=45
+POKENAME=vileplume
+```
+
+tracetest test run -d path/to/test.yaml -e path/to/local.env -w
+
+If you use the .env approach, a new environment will be created in Tracetest if it does not exist where the environment name and id will be the file name without the suffix .env. So, local.env becomes local.
```
Comment (jfermi) [inline docs/docs/cli/running-tests.md:209]:
```suggestion
If you use the .env approach, a new environment will be created in Tracetest. If it does not exist, the environment name and id will be the file name without the suffix .env. So, local.env becomes local.
```

File: docs/docs/cli/running-tests.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+
+The second approach is very useful if you are running tests from a CI pipeline
```
Comment (jfermi) [inline docs/docs/cli/running-tests.md:211]:
```suggestion
The second approach is very useful if you are running tests from a CI pipeline.
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Pull request merged**